app/views/taxonomies/edit.html.erb

```diff
@@ -1,4 +1,4 @@
-<% title _("Edit %s") % @taxonomy.title %>
+<% edit_title @taxonomy, page_header: nil, attrbiture: :title %>
```

Comment:
since you are using kwargs, you don't need to pass all the options - just the ones you don't want the default for.